packages/dev/s2-docs/pages/react-aria/dnd.mdx

```diff
@@ -38,14 +36,22 @@ This example provides items as plain text, HTML, and a custom app-specific data
 ```tsx render files={['packages/dev/s2-docs/pages/react-aria/PokemonCOMPONENT.tsx', 'packages/dev/s2-docs/pages/react-aria/DroppableCOMPONENT.tsx']}
 "use client";
 import {useDragAndDrop} from 'react-aria-components';
-import {PokemonCOMPONENT, Pokemon} from './PokemonCOMPONENT';
-import {DroppableCOMPONENT} from './DroppableCOMPONENT';
+import {PokemonListBox as PokemonCOMPONENT} from './PokemonListBox';
+import {DroppableListBox as DroppableCOMPONENT} from './DroppableListBox';
```

Comment:
Don't these need to state PokemonCOMPONENT so that all the different examples work? I think this will make them all list boxes.